### PR TITLE
Remove the query string parameters from the marketing url

### PR DESCRIPTION
**Pull request by schenedx**
ECOM-6710

@clintonb @MatthewPiatetsky @umar-nawaz @ahsan-ul-haq Please review

### CODE REVIEW

File: sailthru/sailthru_content/services/sailthru_translation_service.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
         seat_priority = ['credit', 'professional', 'verified', 'no-id-professional', 'audit', 'honor']
         sailthru_content_vars = {
             'course_run': True,
-            'marketing_url': url,
+            'marketing_url': url.split('?', 1)[0],
```
Comment (clintonb) [inline sailthru/sailthru_content/services/sailthru_translation_service.py]: Use the `exclude_utm=1` parameter on the API. See https://prod-edx-discovery.edx.org/api-docs/#!/api/Course_list.

Comment (schenedx) [inline sailthru/sailthru_content/services/sailthru_translation_service.py]: Huh, I'll try this out.

**Comment by schenedx** [comment]
@clintonb I took in your feedback, please review again

File: sailthru/requirements/test.txt [hunk 2]
```diff
@@ -4,3 +4,4 @@
 ddt==1.0.0
 factory_boy==2.7.0
 responses==0.5.1
+pytest==3.0.5
```
Comment (clintonb) [inline sailthru/requirements/test.txt]: Alphabetize.

**Pull request merged**